ascii-dev/mt-react
# #160934136 Integrate TravisCI

[Finishes #160934136]
#### What does this PR do?
To configure TravisCI for the application.
#### Description of Task to be completed?
- create travis config
- setup enzyme and jest to run test
- create sample test
#### How should this be manually tested?
After cloning the repo and running `npm install`
You should see the `.travis.yml` file in the project
#### What are the relevant pivotal tracker stories?
[#160934136](https://www.pivotaltracker.com/story/show/160934136)
#### Questions
N/A
